### PR TITLE
TS-19-v2-38: Create constants for all translation labels

### LINKED ISSUE #38: Refactor: Create constants for all translation labels
Acceptance Criteria:
Creates constants for all translation labels and divide files depending on the use area.

- [x] create constants which are common;
- [x] create constants for form elements;
- [x] create constants for service messages;
- [x] create constants for validation messages;

Estimate: 2 day

### PR DESCRIPTION
**Pull request by ViktoriiaKosovan**

### CODE REVIEW

**Review by HiroFive — APPROVED** [review]

File: frontend/src/services/groupService.js [hunk 1]
```diff
@@ -116,11 +116,13 @@ export const removeGroupCardService = (groupId) => {
 };
 
 export const setDisabledGroupService = (group) => {
-    group.disable = true;
+    const bufferGroup = group;
+    bufferGroup.disable = true;
     updateGroupService(group);
```
Comment (Akr1s) [inline frontend/src/services/groupService.js]: technically, `group` and `bufferGroup` link to the same object, but `update...(bufferGroup)` seems more obvious

File: frontend/src/services/groupService.js [hunk 2]
```diff
@@ -116,11 +116,13 @@ export const removeGroupCardService = (groupId) => {
...
 };
 
 export const setEnabledGroupService = (group) => {
-    group.disable = false;
+    const bufferGroup = group;
+    bufferGroup.disable = false;
     updateGroupService(group);
```
Comment (Akr1s) [inline frontend/src/services/groupService.js]: the same as above

File: frontend/src/services/roomService.js [hunk 3]
```diff
@@ -78,12 +78,14 @@ const put = (values) => {
 };
 
 export const setDisabledRoomsService = (room) => {
-    room.disable = true;
+    const bufferRoom = room;
+    bufferRoom.disable = true;
     put(room);
```
Comment (Akr1s) [inline frontend/src/services/roomService.js]: the same like in the previous file

File: frontend/src/services/roomService.js [hunk 4]
```diff
@@ -78,12 +78,14 @@ const put = (values) => {
...
 };
 
 export const setEnabledRoomsService = (room) => {
-    room.disable = false;
+    const bufferRoom = room;
+    bufferRoom.disable = false;
     put(room);
```
Comment (Akr1s) [inline frontend/src/services/roomService.js]: as above

File: frontend/src/services/semesterService.js [hunk 5]
```diff
@@ -256,12 +255,14 @@ export const setDefaultSemesterById = (dataId) => {
 };
 
 export const setDisabledSemestersService = (semester) => {
-    semester.disable = true;
+    const bufferSemester = semester;
+    bufferSemester.disable = true;
     putSemester(semester);
```
Comment (Akr1s) [inline frontend/src/services/semesterService.js]: +1

File: frontend/src/services/semesterService.js [hunk 6]
```diff
@@ -256,12 +255,14 @@ export const setDefaultSemesterById = (dataId) => {
...
 };
 
 export const setEnabledSemestersService = (semester) => {
-    semester.disable = false;
+    const bufferSemester = semester;
+    bufferSemester.disable = false;
     putSemester(semester);
```
Comment (Akr1s) [inline frontend/src/services/semesterService.js]: +1

File: frontend/src/services/subjectService.js [hunk 7]
```diff
@@ -103,11 +103,13 @@ export const removeSubjectCardService = (subjectId) => {
 };
 
 export const setDisabledSubjectsService = (subject) => {
-    subject.disable = true;
+    const bufferSubject = subject;
+    bufferSubject.disable = true;
     updateSubjectService(subject);
```
Comment (Akr1s) [inline frontend/src/services/subjectService.js]: +1

File: frontend/src/services/subjectService.js [hunk 8]
```diff
@@ -103,11 +103,13 @@ export const removeSubjectCardService = (subjectId) => {
...
 };
 
 export const setEnabledSubjectsService = (subject) => {
-    subject.disable = false;
+    const bufferSubject = subject;
+    bufferSubject.disable = false;
     updateSubjectService(subject);
```
Comment (Akr1s) [inline frontend/src/services/subjectService.js]: +1

File: frontend/src/services/teacherService.js [hunk 9]
```diff
@@ -149,11 +149,13 @@ export const removeTeacherCardService = (id) => {
 };
 
 export const setDisabledTeachersService = (teacher) => {
-    teacher.disable = true;
+    const bufferTeacher = teacher;
+    bufferTeacher.disable = true;
     updateTeacherService({ teacher });
```
Comment (Akr1s) [inline frontend/src/services/teacherService.js]: +1

File: frontend/src/services/teacherService.js [hunk 10]
```diff
@@ -149,11 +149,13 @@ export const removeTeacherCardService = (id) => {
...
 };
 
 export const setEnabledTeachersService = (teacher) => {
-    teacher.disable = false;
+    const bufferTeacher = teacher;
+    bufferTeacher.disable = false;
     updateTeacherService({ teacher });
```
Comment (Akr1s) [inline frontend/src/services/teacherService.js]: +1

**Review by Akr1s — CHANGES_REQUESTED** [review]

**Review by Akr1s — APPROVED** [review]

**Review by Niknikson — DISMISSED** [review]

**Review by Niknikson — APPROVED** [review]

**Pull request closed without merging**